Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.5-slim
+
+RUN apt-get update && apt-get install -y \
+	g++ \
+	gfortran \
+	libblas-dev \
+	liblapack-dev \
+	libopenblas-dev \
+	python3-dev \
+	enchant \
+	redis-server \
+    build-essential \
+    npm \
+    nodejs
+
+RUN npm install -g eslint
+
+COPY . /ores
+WORKDIR /ores
+
+RUN pip install pip --upgrade
+RUN pip install wheel
+RUN pip install nltk
+RUN pip install -r /ores/requirements.txt
+RUN pip install -r /ores/test-requirements.txt
+RUN pip install codecov pytest-cov
+RUN python -m nltk.downloader stopwords
+
```

Comment:
Here are the lines to run a service suitable for testing,
```
  ./utility applications.wsgi --config ./config/celery-test/ &
  ./utility applications.celery --config ./config/celery-test/ &
```
It looks like this is bad Docker form, however, since each container is supposed to be running at most one service.  What are your thoughts?